src/base/bittorrent/torrentcreatorthread.h

```diff
@@ -29,17 +29,32 @@
 #ifndef BITTORRENT_TORRENTCREATORTHREAD_H
 #define BITTORRENT_TORRENTCREATORTHREAD_H
 
+#include <libtorrent/version.hpp>
+
 #include <QStringList>
 #include <QThread>
 
 namespace BitTorrent
 {
+#if (LIBTORRENT_VERSION_NUM >= 20000)
+    enum class TorrentFormat
```

Comment:
Will it be used in other places? Should you declare it in a separate header instead?

Reply:
>Will it be used in other places? 

It seems unlikely to me. The enum is really intended for creating torrents only.